src/github.com/matrix-org/dendrite/clientapi/auth/auth.go

```diff
@@ -27,6 +29,13 @@ import (
 	"github.com/matrix-org/util"
 )
 
+// UnknownDeviceID is the default device id if one is not specified.
+var UnknownDeviceID = "unknown-device"
```

Comment:
Is this specified somewhere?
Is it okay for multiple clients to share a device ID?